tpch/execute.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import argparse
+from importlib import import_module
+from pathlib import Path
+
+import pandas as pd
+import polars as pl
+import pyarrow.parquet as pq
+
+pd.options.mode.copy_on_write = True
+pd.options.future.infer_string = True
+
+DATA_DIR = Path("data")
+LINEITEM_PATH = DATA_DIR / "lineitem.parquet"
+REGION_PATH = DATA_DIR / "region.parquet"
+NATION_PATH = DATA_DIR / "nation.parquet"
+SUPPLIER_PATH = DATA_DIR / "supplier.parquet"
+PART_PATH = DATA_DIR / "part.parquet"
+PARTSUPP_PATH = DATA_DIR / "partsupp.parquet"
+ORDERS_PATH = DATA_DIR / "orders.parquet"
+CUSTOMER_PATH = DATA_DIR / "customer.parquet"
+
+BACKEND_READ_FUNC_MAP = {
+    # "pandas": lambda x: pd.read_parquet(x, engine="pyarrow"), # noqa: ERA001
+    "pandas[pyarrow]": lambda x: pd.read_parquet(
+        x, engine="pyarrow", dtype_backend="pyarrow"
+    ),
+    # "polars[eager]": lambda x: pl.read_parquet(x),
+    "polars[lazy]": lambda x: pl.scan_parquet(x),
+    "pyarrow": lambda x: pq.read_table(x),
+    # "dask": lambda x: dd.read_parquet(x, engine="pyarrow", dtype_backend="pyarrow"), # noqa: ERA001
```

Comment:
Should we enable dask again? They fixed the issue for which we removed it in the first place.

Reply:
you are right. tested locally, it seems to be fine 👀